src/client/java/minicraft/level/tile/Tiles.java

```diff
@@ -76,6 +76,9 @@ public static void initTileList() {
 		tiles.put((short)47, new BossWallTile());
 		tiles.put((short)48, new BossFloorTile());
 		tiles.put((short)49, new BossDoorTile());
+		tiles.put((short)50, new FenceTile(Tile.Material.Wood));
+		tiles.put((short)51, new FenceTile(Tile.Material.Stone));
+		tiles.put((short)52, new FenceTile(Tile.Material.Obsidian));
```

Comment:
Using there ids two times is absolutely catastrophic!

Reply:
You are aware that this is because that it was made using another outdated version of the code therefore it’s technically correct at that time. However now it has to be adjusted

Reply:
The point of a merge is to adjust the written code to reflect changes upstream. It wouldn't be hard to fix.